docs/reference/operator_parameters.md

```diff
@@ -12,7 +12,7 @@ configuration.
 
 * CRD-based configuration.  The configuration is stored in the custom YAML
   manifest, an instance of the custom resource definition (CRD) called
-  `postgresql-operator-configuration`.  This CRD is registered by the operator
+  `OperatorConfiguration`.  This CRD is registered by the operator
```

Comment:
This renaming was necessary because the older name with dashes broke the code generation process.